Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -135,67 +135,87 @@ void doDepositionShapeN(const GetParticlePosition& GetPosition,
             // x direction
             // Get particle position after 1/2 push back in position
 #if (defined WARPX_DIM_RZ)
-            const amrex::Real xmid = (rpmid - xmin)*dxi;
+            const double xmid = (rpmid - xmin)*dxi;
```

Comment:
please don't forget to add comments before explicitly selected precision lines :)

Reply:
Comments added!